modules/ROOT/pages/index.adoc

```diff
@@ -13,25 +14,28 @@ image::application-network.png[Application Network]
 
 The connections between nodes are based on traffic between applications or APIs that occurred in the last seven days. If an application appears in multiple workers, they appear as two distinct nodes in the graph.
 
-Anypoint Visualizer collects data from Mule applications, APIs, and proxies deployed to CloudHub or standalone Mule runtime engines (Mules) to discover all incoming and outgoing connections through an embedded plugin. As long as the application, API, or proxy is running on a supported Mule, it is discovered along with any associated dependencies. See xref:technical.adoc[How Anypoint Visualizer Collects Application Data] for additional information on how Anypoint Visualizer collects data displayed in the graph.
+Anypoint Visualizer collects data from Mule applications, APIs, and proxies deployed to CloudHub, standalone Mule runtime engines (Mules), or Runtime Fabric to discover all incoming and outgoing connections through an embedded plugin. As long as the application, API, or proxy is running on a supported Mule, it is discovered along with any associated dependencies. See xref:technical.adoc[How Anypoint Visualizer Collects Application Data] for additional information on how Anypoint Visualizer collects data displayed in the graph.
+
 
-From Anypoint Visualizer you can customize the view arrange applications in layers, inspect their operational monitoring information, and customize the labels displayed for nodes and connections. See xref:customize-your-canvas.adoc[Customize the Canvas] for more information.
+Understanding these dependencies has powerful implications — especially when you have the ability to overlay additional data about these apps onto the network diagram.The power of Visualizer, and the visualizations tailored for Architecture, Troubleshooting and Policies, is understanding this data in the context of your application network. From Anypoint Visualizer, you can inspect performance metrics or policy information, define views, arrange applications in layers, and customize the labels displayed for nodes and connections. See xref:customize-your-canvas.adoc[Customize the Canvas] for more information.
```

Comment:
Missing a space before "The power of Visualizer"

Reply:
```suggestion
Understanding these dependencies has powerful implications — especially when you have the ability to overlay additional data about these apps onto the network diagram. The power of Visualizer, and the visualizations tailored for Architecture, Troubleshooting and Policies, is understanding this data in the context of your application network. From Anypoint Visualizer, you can inspect performance metrics or policy information, define views, arrange applications in layers, and customize the labels displayed for nodes and connections. See xref:customize-your-canvas.adoc[Customize the Canvas] for more information.
```